tests/smoke/support/page/files/allFiles.ts

```diff
@@ -142,6 +154,7 @@ export class AllFilesPage {
     await page.click('.oc-files-actions-rename-trigger')
     await page.fill('.oc-text-input', name[0])
     await page.click('.oc-modal-body-actions-confirm')
+    await page.waitForSelector(`//*[@data-test-resource-name="${name[0]}"]`)
     await page.reload()
```

Comment:
please try to get rid of reload, if we had flakyness here we need to identify that weak part

Reply:
deleted row